.github/workflows/build_linux_pytorch_wheels.yml

```diff
@@ -91,6 +91,14 @@ jobs:
       - name: Create pip cache directory
         run: mkdir -p /tmp/pipcache
 
+      - name: Determine optional arguments
+        if: ${{ inputs.rocm_version }}
+        run: |
+          sdk_version="--rocm-sdk-version \"==${{ inputs.rocm_version }}"\"
+          rocm_suffix=`echo ${{ inputs.rocm_version }} | sed 's/+/-/'`
+          version_suffix="--version-suffix \"+rocm${rocm_suffix}"\"
+          echo optional_arguments="${sdk_version} ${version_suffix}" >> $GITHUB_ENV
```

Comment:
I'd prefer for this to be in a Python script. This style of piping output from one Unix tool to another is harder to work with on Windows (I _do_ have `sed` from git bash, but it's awkward to use...)

We can also add unit tests for such a python script. I can't see through the commands here to know what the expected inputs and outputs are.

Reply:
I'll start the count down until Scott finds out this is actually worth it to script and ask you to flip it :)

Reply:
I moved it to a Python script :smile: This has the advantage that we can easily check if the version string is a valid version.